bitnami/postgresql/Chart.yaml

```diff
@@ -35,4 +35,4 @@ maintainers:
 name: postgresql
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql
-version: 12.8.3
+version: 12.8.4
```

Comment:
@jl-beast When adding a new variable like this, I consider it more appropriate to increase the version in a minor and not in a patch.

Reply:
@corico44 I have bumped it.